deploy/stripe-ct/action.yml

```diff
@@ -0,0 +1,83 @@
+name: Deploy Storefront
+description: Builds Middleware app docker image
+inputs:
+  project_name:
+    description: "Project name"
+    required: true
+  docker_registry_url:
+    description: "Docker registry url"
+    required: false
+    default: "registry.vuestorefront.cloud"
+  cloud_username:
+    description: "Cloud username"
+    required: true
+  cloud_password:
+    description: "Cloud password"
+    required: true
+  cloud_region:
+    description: "Cloud region"
+    required: true
+  console_api_url:
+    description: "URI for Vue Storefront Console API"
+    required: false
+    default: "https://api.platform.vuestorefront.io"
+  version:
+    description: "Version of the app"
+    required: false
+  extension_module_name:
+    description: "Stripe CT extension module name"
+    required: false
+    default: "ct-stripe-extension"
+  extension_module_port:
+    description: "Stripe CT extension module port"
+    required: false
+    default: "8080"
+  notification_module_name:
+    description: "Stripe CT notification module name"
+    required: false
+    default: "ct-stripe-notification"
+  notification_module_port:
+    description: "Stripe CT notification module port"
+    required: false
+    default: "8081"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: chrnorm/deployment-action@releases/v2
+      name: Create GitHub deployment
+      id: deployment
+      with:
+        token: "${{ github.token }}"
+        environment-url: https://${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io
+        environment: production
+        initial-status: in_progress
+
+    - name: Deploy Stripe CT modules (${{ inputs.project_name }}.${{ inputs.cloud_region }}.gcp.storefrontcloud.io)
```

Comment:
I'm just wondering — WDYT about having this on extra switch in the deploy action?

And second thought — How about middleware is current configuration? For example: I have a unified on CT + stripe - Should I use both action to deploy those apps? Or we should add the middleware here as well?